client/shipping-label/state/reducer.js

```diff
@@ -334,16 +334,19 @@ reducers[ OPEN_ADD_ITEM ] = ( state ) => {
 reducers[ CLOSE_ADD_ITEM ] = ( state ) => {
 	return {
 		...state,
-		movedItemIndex: -1,
+		addedItems: null,
 		showAddItemDialog: false,
 	};
 };
 
-reducers[ SET_ADDED_ITEM ] = ( state, { sourcePackageId, movedItemIndex } ) => {
+reducers[ SET_ADDED_ITEM ] = ( state, { sourcePackageId, movedItemIndex, added } ) => {
 	return {
 		...state,
-		sourcePackageId,
-		movedItemIndex,
+		addedItems: { ...state.addedItems,
+			[ sourcePackageId ]: { ...( state.addedItems || {} )[ sourcePackageId ],
```

Comment:
Shouldn't this be checking `state.addedItems[ sourcePackageId ]` instead?

`addedItems` should be added to the initial state here: https://github.com/Automattic/woocommerce-services/blob/748e310eba1fa1c0a804c535b213955c56cd9d11/client/shipping-label/index.js#L31